indy-did-driver/src/main.rs

```diff
@@ -92,8 +93,26 @@ fn init_resolvers(source: &str) -> Resolvers {
                     ),
                 };
 
-                let pool_builder = PoolBuilder::default().transactions(txns).unwrap();
-                let pool = pool_builder.into_shared().unwrap();
+                let pool_builder = PoolBuilder::default()
+                    .transactions(genesis_txns.clone())
+                    .unwrap();
+                let mut pool = pool_builder.into_shared().unwrap();
+
+                // Refresh pool to get current validator set
+                let (txns, _timing) = block_on(perform_refresh(&pool)).unwrap();
```

Comment:
I am wondering about this unwarp on startup: if we don't have connectivity this will panic and kill main.
This would be fixed once we move this into its own thread to periodically refresh the pools and seems to be fine for now to enforce pool updates on startup?